etc/scripts/clean-jupyterlab.sh

```diff
@@ -51,10 +51,6 @@ then
   echo ">>> Installing nbdime 1.1.0"
   pip install --upgrade nbdime==1.1.0
   pip install --upgrade jupyterlab-git==0.10.1
-else
-  echo ">>> Installing nbdime"
-  pip install --upgrade nbdime
-  pip install --upgrade jupyterlab-git==0.20.0rc0
 fi
```

Comment:
I do believe we need that, as git is still rc and considered pre-release.

Reply:
the rc is specified in both Makefile and setup.py so it works without this

Reply:
I don't really like having extra install steps in the clean script. If we've written our build scripts correctly they should install the correct version of nbdime and git, making this install redundant for elyra, but also inconvenient if used for standard lab dev (as I also use it) since you have to uninstall nbdime and git after running it for a clean install. I feel like this section is an artifact of how poor our old build scripts were. Our current build scripts will install the correct version of nbdime and git to match that version of elyra (at least for 0.10.0 and 0.11.0).

tl:dr we should remove this whole if block